docs/changelog.rst

```diff
@@ -1,5 +1,53 @@
 Changelog
 ================================
+CBC SDK 1.5.0 - Released (TBD)
+--------------------------------------
+
+**Breaking Changes:**
+
+* Alerts V7: This is a new version of the Alerts API with certain changes that are not compatible with code written
+  to the old V6 API. For details, please see the :ref:`Alert Migration Guide <alert-migration-guide>`.
```

Comment:
link syntax needs a tweak. replace
:ref:`Alert Migration Guide <alert-migration-guide>`
with
`Alert Migration Guide <alert-migration-guide>`_

Reply:
This is link syntax for RST, not Markdown.